test/check-machines-lifecycle

```diff
@@ -27,11 +27,16 @@ sys.path.append(os.path.join(TEST_DIR, "common"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 
 from machineslib import VirtualMachinesCase  # noqa
-from testlib import nondestructive, skipImage, test_main, wait  # noqa
+from testlib import skipImage, test_main, wait  # noqa
+from machine_core.constants import TEST_OS_DEFAULT  # noqa
```

Comment:
I don't think it nees noqa, not the next line.

Reply:
If there is not a #noqa, `make check` gives error `test/check-machines-lifecycle:32:1: E402 module level import not at top of file`, applicable for both of them.